### PR TITLE
Additional testcafe tests

### PR DESCRIPTION
**Pull request by rogerl50**
Added Bhukhar, Mancala, Dots, Reversi to public library testing. These are all tests that exercise a significant amount of custom JSON code.

### CODE REVIEW

File: tests/testcafe/tests.js [hunk 1]
```diff
@@ -300,18 +275,62 @@ test('Dynamic expressions', async t => {
...
+
 publicLibraryButtons('Blue',               0, '096bdf3bd07bb277c2c1f4cc132f0695', [
   'fcc3fa2c-c091-41bc-8737-54d8b9d3a929', 'd3ab9f5f-daa4-4d81-8004-50a9c90af88e_incrementButton',
   'd3ab9f5f-daa4-4d81-8004-50a9c90af88e_incrementButton', 'd3ab9f5f-daa4-4d81-8004-50a9c90af88e_decrementButton',
   'reset_button', 'buttonInputGo', 'fcc3fa2c-c091-41bc-8737-54d8b9d3a929', '9n2q'
 ]);
+publicLibraryButtons('Dots',               0, '23894df38f786cb014fa1cd79f2345db', [ 'reset', 'buttonInputGo', 'col11', 'col21', 'col12', 'col22', 'row11', 'row31', 'row21', 'row32', 'row12', 'row42', 'row22', 'row23', 'col23' ]);
+publicLibraryButtons('Reversi',            0, '35e0017570f9ecd206a2317c1528be36',
+       [
+         [ ()=>{return Selector("#zpiece15")}, ()=>{return Selector("#sq23")} ],
```
Comment (ArnoldSmith86) [inline tests/testcafe/tests.js:323]: `()=>{return Selector("#zpiece15")}` should be the same as `()=>Selector("#zpiece15")` (note the missing braces). Just FYI.

Comment (rogerl50) [reply tests/testcafe/tests.js]: Thanks. Good to know. Fixed.

**Review by ArnoldSmith86 — APPROVED** [review]
I can't see problems and it is very nice to have more tests.

Would be nice to restore the alphabetical order but :man_shrugging:.

**Review by 96LawDawg — APPROVED** [review]
Approved.  The tests pass, so this is obviously working.

**Pull request merged**